keras_nlp/applications/bert.py

```diff
@@ -52,6 +52,7 @@ class BertModel(keras.Model):
             two-layer feedforward network for each transformer.
         initializer_range: The initialzer range to use for a truncated normal
```

Comment:
This feels kind of limiting as an API. Is there a way we can expose this as a normal initializer? And just default it to truncated normal with 0.02?

There's a lot of research on better initializers for deeper/more stable transformers, keeping a real initializer argument here could allow for easier experimentation.